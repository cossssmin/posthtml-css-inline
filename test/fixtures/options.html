<!DOCTYPE html>
<html>
<head>
  <style>
    .text-sm {
      font-size: 12px;
    }

    .text-blue {
      color: blue;
    }

    div {
      display: flex;
    }

    div {
      font-size: 14px !important;
    }

    @media (min-width: 640px) {
      .text-sm {
        font-size: 16px;
      }
    }
  </style>
</head>
<body>
  <div class="text-blue text-sm" data-attr-no-quotes>text-sm</div>
</body>
</html>
