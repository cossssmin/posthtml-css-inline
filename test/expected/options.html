<!DOCTYPE html>
<html>
<head>
  <style>@media (min-width: 640px) {
      .text-sm {
        font-size: 16px;
      }
    }</style>
</head>
<body>
  <div class="text-blue text-sm" data-attr-no-quotes style="font-size: 14px !important; color: blue; display: flex">text</div>
</body>
</html>
